jenkins/opensearch_dashboards/Jenkinsfile

```diff
@@ -0,0 +1,143 @@
+pipeline {
+    agent none
+    environment {
+        OPENSEARCH_BUILD_ID = "${BUILD_NUMBER}"
```

Comment:
OPENSEARCH_DASHBOARDS_BUILD_ID?

Reply:
Alternative proposal, keep it as BUILD_NUMBER